xbmc/windowing/wayland/WinSystemWaylandWebOS.h

```diff
@@ -48,13 +49,18 @@ class CWinSystemWaylandWebOS : public CWinSystemWayland
   std::unique_ptr<KODI::WINDOWING::IOSScreenSaver> GetOSScreenSaverImpl() override;
 
 private:
+  static bool OnAppLifecycleEvent(LSHandle* sh, LSMessage* reply, void* ctx);
```

Comment:
```suggestion
  static bool OnAppLifecycleEventWrapper(LSHandle* sh, LSMessage* reply, void* ctx);
```
the functions are a bit hard to discertain - just a suggestion though

Reply:
I agree, this is just a wrapper around the C-Style API